.github/workflows/wine-arch.yml

```diff
@@ -0,0 +1,29 @@
+name: Wine Archlinux CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: archlinux:latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Compilation
+        run: |
+          echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist" >> /etc/pacman.conf
+          yes|pacman -Syu --noconfirm base-devel sudo
```

Comment:
Does this actually work, without running `pacman-key` as described [here](https://hub.docker.com/_/archlinux)?

Reply:
It's working, I can add the key if it really necessary.

Reply:
Okay, as long as it works that should be okay.